libraries/AP_RPM/AP_RPM_Params.cpp

```diff
@@ -62,8 +62,8 @@ const AP_Param::GroupInfo AP_RPM_Params::var_info[] = {
     AP_GROUPINFO("PIN", 6, AP_RPM_Params, pin, -1),
 
     // @Param: ESC_MASK
-    // @DisplayName: Bitmask of ESC telemetry channels to average
-    // @Description: Mask of channels which support ESC rpm telemetry. RPM telemetry of the selected channels will be averaged
+    // @DisplayName: Bitmask of ESC telemetry channels
```

Comment:
As a side note re naming, we should drop words like "Bitmask" from the Display Name and should start with the subsystem (e.g. "RPM").  The Display Name is the top level name of the parameter shown to the user so it really should be something like, "RPM ESC telemetry channels".